app/src/app-settings.cpp

```diff
@@ -20,17 +20,16 @@ AppSettings::AppSettings()
     memcpy(wifiDefault.ssid, "admin", strlen("admin"));
     memcpy(wifiDefault.pass, "admin", strlen("admin"));
     // Assign defaults
-    
+
     memset(&defaultSettings.name, 0, sizeof(defaultSettings.name));
     memcpy(&defaultSettings.name, "MUX_DEFAULT", strlen("MUX_DEFAULT"));
     defaultSettings.wifi0 = wifiDefault;
     defaultSettings.wifi1 = wifiDefault;
     defaultSettings.wifi2 = wifiDefault;
 }
 
-
 //------------------------------------------------------------------------------
-AppSettings&  AppSettings::instance()
+AppSettings &AppSettings::instance()

```

Comment:
you are changing the code format in many places. Don't do that, leave the code format as it is and develop your  changes in the same format

Reply:
It should be fixed right now, my bad